code/modules/plumbing/plumbers/bottler.dm

```diff
@@ -0,0 +1,79 @@
+/obj/machinery/plumbing/bottler
+	name = "chemical bottler"
+	desc = "Puts reagents into containers, like bottles and beakers."
+	icon_state = "bottler"
+	layer = ABOVE_ALL_MOB_LAYER
+	reagent_flags = TRANSPARENT | DRAINABLE
+	rcd_cost = 50
+	rcd_delay = 50
```

Comment:
you made it worse

https://github.com/tgstation/tgstation/blob/9db5b7acf608cb850868074ea13e6d655a7a1b97/code/modules/plumbing/plumbers/synthesizer.dm#L8-L9

Reply:
its a big machine

Reply:
ok, I just wanted to know if the original values were intended or not lol...

Reply:
then say it

Reply:
I did...

>  is that intended?

Reply:
ok shaggy avatar